### PR TITLE
Add support for exporting images from DeploymentConfig

### LINKED ISSUE #41: Images from DeploymentConfig are not exported

### PR DESCRIPTION
**Pull request by kadel**
fixes #41

### CODE REVIEW

**Comment by kadel** [comment]
ping @surajssd

File: openshift2nulecule/utils.py [hunk 1]
```diff
@@ -139,13 +139,12 @@ def replace_registry_host(image_name, new_registry):
...
-    Checks if image specified in ReplicationController is from internal
-    registry. If it is from internal registry....
-    of registry
+    Checks if image specified in ReplicationController or DeploymentConfig
+    is from internal OpenShift registry.
     TODO: support for Pod
 
     Args:
-       obj (dict): ReplicationController
+       obj (dict): ReplicationController or ReplicationController
```
Comment (surajssd) [inline openshift2nulecule/utils.py]: Typo: Both are same things ! :D

Comment (kadel) [inline openshift2nulecule/utils.py]: Oy :relaxed: , thx :wink:

**Comment by surajssd** [comment]
Other part is good.

**Pull request merged**